### PR TITLE
【prim】Slice grad

### PR DESCRIPTION
**Pull request by xiaoguoguo626807**
### PR types
Others 

### PR changes
Others

### Describe
slice grad 拆解规则

### CODE REVIEW

File: tools/check_file_diff_approvals.sh [hunk 1]
```diff
@@ -212,7 +212,6 @@ for API_FILE in ${API_FILES[*]}; do
       elif [ "${API_FILE}" == "python/paddle/incubate/autograd/primitives.py" ] || [ "${API_FILE}" == "python/paddle/incubate/autograd/composite_rules.py" ]; then
             echo_line="You must have one RD (cyber-pioneer(chenzhuo), JiabinYang) approval for changing ${API_FILE} , which manages the composite rules.\n"
             check_approval 1 cyber-pioneer JiabinYang
-
```
Comment (JiabinYang) [inline tools/check_file_diff_approvals.sh:215]: why this?

**Review by JiabinYang — APPROVED** [review]
LGTM

File: tools/check_file_diff_approvals.sh (hunk 1 above)
Comment (xiaoguoguo626807) [reply tools/check_file_diff_approvals.sh:215]: modify code style

**Review by zhiqiu — APPROVED** [review]
LGTM

**Pull request merged**